superset/assets/src/dashboard/components/FilterIndicatorsContainer.jsx

```diff
@@ -150,21 +150,16 @@ export default class FilterIndicatorsContainer extends React.PureComponent {
     }
 
     const indicators = this.getFilterIndicators();
-    // if total indicators <= 5, show all
-    // else: show top 4 indicators, and show all the rest in group
+    // if total indicators <= FILTER_INDICATORS_DISPLAY_LENGTH,
+    // show indicator for each filter field.
+    // else: show single group indicator.
     const showExtraIndicatorsInGroup =
```

Comment:
rename to `showIndicatorsInGroup` since we're showing all of them there, not just the extra ones

Reply:
fixed.